api/src/main/resources/workbench-api.yaml

```diff
@@ -5345,6 +5385,18 @@ definitions:
         format: int64
         description: 'Timestamp when the user first changed from Unregistered to Registered
           data access level. May be null if the user has never done so.'
+      rasLinkLoginGovName:
+        type: string
+        description: The user's identity name in login.gov.
+      rasLinkLoginGovExpireTime:
+        type: integer
+        format: int64
+        description: The user's login.gov/AoU link expiration time
+      rasLinkLoginGovCompletionTime:
```

Comment:
note: eventually we will need a bypass time for this as well. Don't know if you want it in this PR or not

Reply:
ah, right, I will just added it now.